doc/digital-receipt/implementation/digital-receipt-implementation.md

```diff
@@ -0,0 +1,667 @@
+| 1  | Navigate to the configuration section and go to Queue  |
+| 2  | Configure Queue  |
+| 3  | Copy the URLs to your local machine (Required for CashBox configuration later)   |
+| 4  | For all countries: Change port to the next free port (+1) and <br/> a.	if no suffix exists after the port: add the suffix "/name_queue" to the URL ("name" can be freely chosen) <br/> b.	if suffix already exists: add the suffix "_queue" to the URL  |
+| 5  | Germany & France only: Change grpc port to the next free port (if port is free no need to go up to the next free port) and add the suffix "/name_queue" to the URL ("name" can be freely chosen)  |
+| 6  | Save changes  |
+
+### Helper 
+
+| Step  | Description |
+| ------------- | ------------- |
+| 1  | Navigate to Helper  |
+| 2  | Create new helper  |
+| 3  | Add description  |
+| 4  | Select  package name "fiskaltrust.service.helper.posapi"   |
+| 5  | Select latest package version   |
+| 6  | Select the outlet of CashBox    |
+| 7  | Save configuration   |
+| 8  | Klick configure helper   |
+| 9  | All Counties: Insert the previously saved Queue URLs to the Helper URLs and add the suffix "/name" to the URL (analogue to the naming in queue configuration). Germany & France only: Add also GRPC URL with next free port and add the suffix "/name" to the URL (analogue to the naming in queue configuration).   |
+| 10  | Save configuration and close   |
+
+### CashBox 
+
+| Step  | Description |
+| ------------- | ------------- |
+| 1  | Navigate to CashBox   |
+| 2  | Select your CashBox and click edit by list  |
+| 3  | Navigate to Helpers  |
+| 4  | Activate the POS API Helper  |
+| 5  | Save configuration  |
+| 6  | Klick rebuild configuration  |
+
+### Restart
+
+Restart the fiskaltrust.Middleware to apply the changes. 
+
+# Create receipts with the print endpoint – receive receipt with response via POS API (preferred) 
+
+The POS API is the latest addition to the digital receipt ecosystem. The POS API is a superset of the Middleware's "original" IPOS interface, and uses the same models for /sign, /journal and /echo. The core features of this API provides a variety of different functionalities for Point of Sales software and is the central entry point to the fiskaltrust.Middleware. For the digital receipt the /print endpoint is required, to digitally print digital receipts.
+
+This means that existing implementations can very easily be reused by adjusting them to the asynchronous flow. The IPOS interface will continue to be fully supported by the Middleware.
+
+As most operations especially /print requests may take an extended amount of time, this API is designed in a completely asynchronous way. After sending a request to one of the endpoints listed below, the API immediately returns a message identifier (i.e. an UID), which can then be used to query for the result via the /response endpoint. Please note: The /sign operation does not necessary needs to be implemented for digital receipt application. 
```

Comment:
Actually mostly the /pay operation will take a lot of time, /print is pretty fast i hope :D